app.js

```diff
@@ -0,0 +1,15 @@
+const express = require('express')
+const app = express()
+const {getAllTopics} = require('./controllers/topics.controllers')
+
+app.use(express.json())
```

Comment:
Has this line of code been driven by a test? Every code written shoul dbe driven by a test or part of a test but we haven't yet written a test that requires express.json() yet